File: JavaScript(NodeJS)/old_approach/old_approach.js
const readline = require('readline');

function reverse_num(number)
{
	let sum = 0;
	let digit = 0;
	while (number != 0)
	{
		digit = parseInt(number % 10);
		sum = sum * 10 + digit;
		number = parseInt(number / 10);
	}

	return sum;
}


const rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout
});

rl.question('Please insert a number: ', (answer) => {
	//input: 123456789098765
	let num = parseInt(answer);
	let res = reverse_num(num);
	console.log(res);

	rl.close(); 
});

File: Julia/old_approach/old_approach.jl
function reversenumber(number::Int64)::Int64
	sum = 0
	digit = 0

	while number != 0
		digit = number % 10
		sum = sum * 10 + digit
		number = convert(Int64, floor(number / 10))
	end

	return sum
end

input = readline()
number = parse(Int64, input)
res = reversenumber(number)
println(res)